developer_setup.md

```diff
@@ -277,6 +288,14 @@ targeted end-user testing. See the [minimal mode guide](developer_setup/minimal_
 bin/update                # Updates dependencies using bundler and bower, runs migrations, prepares test db.
 ```
 
+### Upgrading tools after an OS upgrade
```

Comment:
This appears to be Mac only.  It sounds like it belongs under either Mac setup or troubleshooting?